vulture/core.py

```diff
@@ -206,7 +207,8 @@ def file_lineno(item):
         unused_item_found = False
         for item in sorted(
                 self.unused_funcs + self.unused_imports + self.unused_props +
-                self.unused_vars + self.unused_attrs, key=file_lineno):
+                self.unused_classes + self.unused_vars + self.unused_attrs,
+                key=file_lineno):
```

Comment:
Should we introduce a new variable `all = self.unused_funcs + self.unused_imports + self.unused_props + self.unused_classes + self.unused_vars + self.unused_attrs` ?